assets/sass/main.scss

```diff
@@ -46,3 +46,4 @@ $font-size-base:  $aplus-font-size-base;
 @import 'components/focus';
 @import 'components/frontpage-cards';
 @import 'protocols/quiz1';
+@import 'components//login';
```

Comment:
These statements should be sorted so that all components are imported before the protocols. In addition, double slash `//`.